.github/ISSUE_TEMPLATE/story.md

```diff
@@ -44,3 +44,49 @@ What else should contributors [keep in mind](https://fleetdm.com/handbook/compan
 1. 
 2. 
 -->
+
+
+## Test plan
```

Comment:
What if this was `## QA` instead of `## Test plan`?

Thinking: Rather than filling this out ahead of time, this could be filled out after the fact in many cases (consider especially the small issues).

Up to Luke